redesign/index.html

```diff
@@ -2,9 +2,207 @@
 <html>
   <head>
     <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="dwyl is a worldwide community of people using technology to solve real problems.">
+    <meta name="author" content="@dwyl">
+    <meta name="keywords" content="do, what, you, love, dwyl, mvp, time, tracking, help, developers, code, frontend, backend, tooling, tools">
+    <meta name="robots" content="index, follow, noarchive">
+    <meta name="base" content="https://www.dwyl.com">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css">
```

Comment:
We might want to consider a customised version of fontawesome at a later stage when we have a back end if we feel there are performance benefits from it.